api/openapi/auth.yml

```
@@ -32,6 +32,305 @@ tags:
       url: http://docs.mainflux.io/
 
 paths:
+  /domains:
+    post:
+      tags:
+        - Domains
+      summary: Adds new domain
+      description: |
+        Adds new domain.
+      requestBody:
+        $ref: "#/components/requestBodies/DomainCreateReq"
+      responses:
+        "201":
+          $ref: "#/components/responses/DomainCreateRes"
+        "400":
+          description: Failed due to malformed JSON.
+        "401":
+          description: Missing or invalid access token provided.
+        "409":
+          description: Failed due to using an existing identity.
```

the unique variable being used at the moment is `alias`?

---

Yes

---

this then should be:

```suggestion
          description: Failed due to using an existing alias.
```